tests/unittest/_torch/sampler/test_torch_sampler.py

```
@@ -2893,3 +2906,213 @@ def test_reject_speculative_draft_tokens(self):
+                max_draft_len=0,
+                max_num_sequences=8,
+                max_beam_width=1,
+                max_total_draft_tokens=0,
+                disable_overlap_scheduler=True,
+            )
+        )
+
+    def _run(self, requests, ngram_sizes, num_steps=None, num_beams=None, sampler=None):
+        num_steps = num_steps or [1] * len(requests)
+        num_beams = num_beams or [1] * len(requests)
+        total_rows = sum(s * b for s, b in zip(num_steps, num_beams))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)